src/turn_integrator.py

```diff
@@ -25,22 +26,42 @@ def turn_credentials(line_name):
 
     return eval_credentials(config_json)
```

Comment:
Should we be handling the `Exception` in the eval_credentials function here, or do you want it to crash in that case?

Reply:
I think bubble up for now, whomever is using the library can choose how to handle this.